### DISCUSSION TITLE
Application for a Media Type

### DISCUSSION THREAD

**Discussion by vanillajonathan** [PRQL/prql#3600] [General]
PRQL could apply for a media type with the IANA.

Ideally we would want `application/prql` (just as there is `application/sql`). However the application form states:
> Registrations in the standards tree must be approved by the IESG or correspond to a formal publication by a recognized standards body. See [RFC 6838](https://www.iana.org/go/rfc6838) (section 3.1).

We don't have any RFC or  any standard document published by a recognized standards body, so unfortunaly having it in the standards tree might not be an option for us at this point in time. It could be filed in the vendor tree (or personal tree) though, which means it would look something like `application/vnd.prql`.

I guess @max-sixty would be the person to file the application with IANA.

https://www.iana.org/form/media-types

**Comment by max-sixty** [comment]
Would it need to be me that submits it? 

If so, I'm happy to hit the button on something if someone wants to draft it in a PR. (to set expectations: I'm unlikely to prioritize filling this out myself)

**Reply by vanillajonathan** [reply]
I don't know, I figured it would be you since you're the maintainer, but I guess @snth or @eitsupi could do it. It ought to be someone with an official PRQL affiliation though since the application needs a contact person for a author or "change controller".
